lib/rules/disallow-arrow-functions.js

```diff
@@ -0,0 +1,67 @@
+/**
+ * Disallows arrow functions.
+ *
+ * Why enable this rule? Arrow functions might cause more problems than they
+ * solve:
+ *
+ * - Object-orientation may be better without ECMAScript's `this`.
+ * - You can't name an arrow function.
+ * - Arrow functions' syntax can cause maintenance issues; see
+ *   `disallowShorthandArrowFunctions`.
+ * - Arrow functions shouldn't be used on prototypes, as objects' methods,
```

Comment:
Yeah, it's correct; though it always looks strange even to native English speakers :)